cmd/operator/README.md

```diff
@@ -78,4 +79,12 @@ $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/
 ```
 
 Now you can deploy the device plugins by creating corresponding custom resources.
-The samples for them are available [here](/deployments/operator/samples/).
\ No newline at end of file
+The samples for them are available [here](/deployments/operator/samples/).
+
+## Known issues
+
+When the operator is run with leader election enabled, that is with the option
+`--enable-leader-election`, make sure the cluster is not overloaded with excessive
+number of pods. Otherwise a heart beat used by the leader election code may trigger
+a timeout and crash. We are going to use different clients for the controller and
+leader election code to alleviate the issue.
```

Comment:
Perhaps add a reference to #476 so that the reader is able to see the progress, like "See more details in #476"

Reply:
Added link to #476